en/docs/cookies/serving.md

```diff
@@ -0,0 +1,267 @@
+
+OneFlow provides services for the model by docking with the [Triton Inference Server](https://github.com/Triton-inference-server/server). 
+
+OneFlow developers can deploy the model directly through Triton after training the model, use the rich features of Triton, such as Dynamic batching, Model Pipelines and HTTP/gRPC interface, and integrate it into online products quickly and efficiently.
+
+Contents of this article：
+
+- Quick Start of OneFlow deployment
+- Introduction to OneFlow Serving Architecture
+- Analysis of OneFlow Process from Training to Deployment
+
+## Quick Start of OneFlow deployment
+
+The project [OneFlow Serving: Neural Style Transfer](https://oneflow.cloud/#/project/public/code?id=2eec2f768cdfe5709dc4c01e389fd65c) has been prepared on OneFlow Cloud. Referring to the description, developers can deploy the project with one click and view the running result.
+
+![](./imgs/oneflow-serving-demo.png)
+
+Analyzing the code, we can find the following key points:
+
+- Triton server and WEB application server are started in `run_cloud.sh`:
+```bash
+/opt/tritonserver/bin/tritonserver --model-store $(pwd)/model_repo > 1.txt &
+  python3 server.py
+```
+
+- `server.py` is just simple and normal URL routing, so `stylize` in `infer.py` does the real inference work. Its result is obtained inside the `stylize` function through interacting with the HTTP and Triton server.
+```python
+def stylize(content_path, output_path, style='udnie'):
+    triton_client = httpclient.InferenceServerClient(url='127.0.0.1:8000')
+    ...
+    inputs.append(httpclient.InferInput('INPUT_0', image.shape, 'FP32'))
+    ...
+    outputs.append(httpclient.InferRequestedOutput('OUTPUT_0', binary_data=True))
+    ...
+```
+
+- Pretrained models are placed under `model_repo`, which is organized according to Triton's conventions.
+
+This simple online example illustrates that how the OneFlow model can be deployed through Triton and how business modules interact with the Triton server to obtain inference results.
+
+If you want to run this example locally, you can also download [demo.zip](https://oneflow-public.oss-cn-beijing.aliyuncs.com/oneflow-documentation/serving/demo.zip), unzip it and run the file `run.sh` in it.
+
+```shell
+bash run.sh
+```
+
+Next we will introduce the detailed process of OneFlow from training to deployment in detail.
+
+## Analysis of OneFlow Process from Training to Deployment
+
+Firstly we can understand the relationship between OneFlow and Triton in general through the following figure.
+
+![](./imgs/triton-oneflow-backend.png)
+
+As can be seen from the above figure, Triton is in the position of connecting the client and OneFlow: it provides HTTP, gRPC and C interfaces, so that developers can flexibly make an inference request and get the result. 
```

Comment:
```suggestion
It can be seen that Triton is in the position of connecting the client and OneFlow: it provides HTTP, gRPC, and C interfaces, so that users can flexibly make an inference request and get the result. 
```